src/DetailsView/reports/components/instance-details.scss

```diff
@@ -8,7 +8,9 @@
     background: $neutral-0;
     border-radius: 4px;
     box-shadow: 0px 0.6px 1.8px rgba(0, 0, 0, 0.108), 0px 3.2px 7.2px rgba(0, 0, 0, 0.132);
-    display: block;
+    display: table;
+    width: -webkit-fill-available;
```

Comment:
should we care about other browsers here? found a stack overflow about it [here](https://stackoverflow.com/questions/26275982/putting-moz-available-and-webkit-fill-available-in-one-width-css-property)